src/sql/parts/modelComponents/dropdown.component.ts

```diff
@@ -14,22 +14,29 @@ import Event, { Emitter } from 'vs/base/common/event';
 import { ComponentBase } from 'sql/parts/modelComponents/componentBase';
 import { IComponent, IComponentDescriptor, IModelStore, ComponentEventType } from 'sql/parts/modelComponents/interfaces';
 import { Dropdown, IDropdownOptions } from 'sql/base/browser/ui/editableDropdown/dropdown';
+import { SelectBox } from 'sql/base/browser/ui/selectBox/selectBox';
 import { CommonServiceInterface } from 'sql/services/common/commonServiceInterface.service';
 import { attachListStyler } from 'vs/platform/theme/common/styler';
-import { attachEditableDropdownStyler } from 'sql/common/theme/styler';
+import { attachEditableDropdownStyler , attachSelectBoxStyler} from 'sql/common/theme/styler';
 
 @Component({
 	selector: 'dropdown',
 	template: `
-		<div #input style="width: 100%"></div>
+
+	<div>
+		<div [style.visibility]="editableVisibility()" [style.position]="editablePosition()"  #editableDropDown style="width: 100%;z-index: 1; top: 0px; left: 0px;"></div>
+		<div [style.visibility]="notEditableVisibility()" [style.position]="notEditablePosition()" #dropDown style="width: 100%;z-index: 2; top: 0px; left: 0px;"></div>
```

Comment:
Is it possible to use style.display instead of style.visibility? That way we don't need to recalculate position

Reply:
yes thanks!